sleap/gui/commands.py

```diff
@@ -286,7 +286,7 @@ def openProject(self, filename: Optional[str] = None, first_open: bool = False):
         """
         self.execute(OpenProject, filename=filename, first_open=first_open)
 
-    def importAT(self):
+    def important(self):
```

Comment:
Everything looks pretty good except for renaming this function

Reply:
sorry, slipped through my fingers. Adding `importat` to skips, and redoing the fixes... force pushed